src/app/components/EnterPage/enter.scss

```diff
@@ -0,0 +1,72 @@
+:local {
+  .enter {
+    display: block;
+    width: 426px;
+    height: 238px;
+    margin-left: 50%;
+    margin-top: 48px;
+    transform: translateX(-50%);
+    background: #ffffff;
+    box-shadow: 0px 0px 12px rgba(188, 188, 188, 0.25);
+  }
+
+  .linkDecoration {
+    text-decoration: none;
+    color: #1c83c6;
+  }
+
+  .titleContent {
+    padding-left: 36px;
+    padding-top: 24px;
+
+    font-family: 'Mulish';
+    font-style: normal;
+    font-weight: 600;
+    font-size: 24px;
+    line-height: 30px;
```

Comment:
Set the default `font-family`, `font-style`, `font-weight`, `font-size`, `line-height`, `color` at the top-level container. The setting will be applied to all descendants. No need to repeat.